docs/user_guide/reductions.rst

```diff
@@ -0,0 +1,263 @@
+The Reductions Approach to Disparity Mitigation
+===============================================
+
+The "Reductions Approach" is named because it *reduces* the problem of training a
+classifier while satisfying a disparity constraint to that of training a classifier
+on weighted data.
+This technique was first described in [#1]_.
+
+
+Fair Classification
+-------------------
+
+Consider the standard setting for binary classification.
+In such as case, we wish to learn the classifer
+:math:`h \in \mathcal{H}` which minimises the classification
+error :math:`\mbox{err}(h) = \P[ h(X) \neq Y]`.
+Now, let us suppose that we can devise a constraint function
+:math:`\mathbf{C}(h)` which measures the violation of our
+disparity constraint.
+We will then want to learn the classifier
+:math:`h \in \mathcal{H}`
+which satisfies
+
+.. math::
+   \min_{h \in \mathcal{H}} \mbox{err}{(h)}
+   \quad
+   \mbox{subject to}
+   \quad
+   \mathbf{C}(h) \le \mathbf{c}
+   :label: eq_fairclassify_base
+
+where :math:`\mathbf{c}` is a (vector) tolerance of the constraint
+violation.
+We will specify the exact form of :math:`\mathbf{C}` in a later section.
+
+For fairness problems, we can make better tradeoffs if we make use of
+*randomized classifiers*.
+A randomized classifier, :math:`Q` consists of a weighted collection
+of classifiers :math:`h \in \mathcal{H}`.
+To make a prediction using :math:`Q`, we first select one of the
+classifiers, :math:`h`, at random (according to their weights) and then
+use that :math:`h` to make the prediction.
+If :math:`\Delta` represents the set of all possible weighted collections
+of classifiers, then we can rewrite :eq:`eq_fairclassify_base` as:
+
+.. math::
```

Comment:
Is there a way to read these maths on GitHub or locally?  or do you have to build the website to see them rendered?  or better to just practice becoming fluent is this markup :)

Reply:
You can build the website (with the [CircleCI YAML](https://github.com/fairlearn/fairlearn/blob/master/.circleci/config.yml#L29) being my go-to reference, or look at the Artifacts on the CircleCI build on the PR.

Fortunately, at least these bits are written in LaTeX, and hence moderately readable.

Reply:
Thanks!  I'll check that out, latex is still new to me :)